elasticsearch/client.go

```diff
@@ -101,6 +107,24 @@ func (cli *Client) post(ctx context.Context, index string, docType string, actio
 	return response, err
 }
 
+//CreateNewEmptyIndex is a method that creates an empty Elasticsearch index with the given indexName
+//It returns true if the index was created successfully, otherwise false
+func (cli *Client) CreateNewEmptyIndex(ctx context.Context, indexName string) (bool, error) {
+	esClient := dpelasticsearch.NewClient(cli.cfg.ElasticSearchAPIURL, cli.cfg.SignElasticsearchRequests, 5)
```

Comment:
Do we need to create the client each time or could this be a field of the `Client`  struct?

Reply:
oops I should have thought of that - thanks for pointing out